fbw-a32nx/src/systems/fmgc/src/flightplanning/new/WaypointEntryUtils.ts

```diff
@@ -60,20 +48,33 @@ export class WaypointEntryUtils {
     /**
      * Parse a place string into a position
      */
-    static async parsePlace(fms: DisplayInterface, place: string): Promise<Waypoint> {
+    static async parsePlace(fms: DisplayInterface, place: string): Promise<Fix> {
         if (WaypointEntryUtils.isRunwayFormat(place)) {
             return WaypointEntryUtils.parseRunway(place);
+        } if (WaypointEntryUtils.isAirportFormat(place)) {
```

Comment:
You can't do this. It will mask all 4 letter waypoints (of which there are several thousand).